apps/evm_family/evm_user_verification.c

```diff
@@ -122,22 +119,24 @@ bool evm_verify_transfer(const evm_txn_context_t *txn_context) {
 #endif
 
   // verify recipient address; TODO: handle harmony address encoding
-  eth_get_to_address(&txn_context->transaction_info, &to_address);
+  eth_get_to_address(txn_context, &to_address);
   ethereum_address_checksum(
       to_address, &address[2], false, g_evm_app->chain_id);
-  if (!core_scroll_page(title, address, evm_send_error)) {
+  snprintf(
+      display, sizeof(display), UI_TEXT_BTC_SEND_PROMPT, unit, g_evm_app->name);
```

Comment:
Either rename the macro or create a new one for this

Reply:
renamed